hdfs/util.py

```diff
@@ -31,6 +31,18 @@ def __init__(self, message, *args, **kwargs):
     self.exception = kwargs.get("exception")
 
 
+def wrapped_consumer(asyncWriter, data):
```

Comment:
Let's make this private and use Python naming conventions (recommend shortening `asyncWriter` to `writer`).